AUTHORS.rst

```diff
@@ -48,145 +48,145 @@ Other contributors
 
 Some other past or present contributors are:
 
-* `Ahmad Chamma`_
+* `Ahmad Chamma`_: INRIA, Saclay, France
 * `Aina Frau-Pascual`_
-* `Alex Rothberg`_
-* `Alexandre Abadie`_
-* `Alexandre Abraham`_
-* `Alexandre Gramfort`_
+* `Alex Rothberg`_: Free Agency, New York, USA
+* `Alexandre Abadie`_: Inria, Paris
+* `Alexandre Abraham`_: Bazemont, Île-de-France, France
+* `Alexandre Gramfort`_: Meta, France
 * `Alexandre Savio`_
-* `Alexis Thual`_
-* `Alisha Kodibagkar`_
+* `Alexis Thual`_: INRIA, Saclay, France
```

Comment:
```suggestion
* `Alexis Thual`_: Inria, Saclay, France
```

Reply:
Note that Alexis has multiple affiliations (INSERM, College de France)

Reply:
yeah there are a few others like that: not sure how to handle it.

also not sure we should make any promise to keep all to those up to date: I feel that "affiliation at contribution time" feels more manageable for maintenance

though obviously if contributors want to drop by and update their affiliation they can do so